Automattic/wp-calypso
# Blog onboarding: Hide use your own domain CTA for now

Related to p1684439743066089/1684426986.377669-slack-CKZHG0QCR

## Proposed Changes

* For the time being lets hide the "Use a domain I own" CTA. See related Slack conversation.

Before | After
--|--
![use-domain](https://github.com/Automattic/wp-calypso/assets/140841/d5cd8b5d-b3ff-43fb-99a8-ff8cd511d70d) | ![use-domain-gone](https://github.com/Automattic/wp-calypso/assets/140841/a2025106-5de5-4810-8413-3f07a459653d)

## Testing Instructions



* Load this PR
* Using a new user OR a user without any site, start the flow with [/setup/start-writing](http://calypso.localhost:3000/setup/start-writing).
* When you get to the choose a domain step, the "Use a domain I own" option should not be present.

## Pre-merge Checklist
